specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2023-06-28-preview/managedccf.json

```diff
@@ -0,0 +1,708 @@
+      "properties": {
+        "appName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Unique name for the Managed CCF."
+        },
+        "appUri": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Endpoint for calling Managed CCF Service."
+        },
+        "identityServiceUri": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Endpoint for accessing network identity."
+        },
+        "memberIdentityCertificates": {
+          "type": "array",
+          "description": "List of member identity certificates for  Managed CCF",
+          "items": {
+            "$ref": "#/definitions/MemberIdentityCertificate"
+          },
+          "x-ms-identifiers": []
+        },
+        "deploymentType": {
+          "description": "Deployment Type of Managed CCF",
+          "$ref": "#/definitions/DeploymentType"
+        },
+        "runningState": {
+          "description": "Object representing RunningState for Managed CCF.",
+          "$ref": "#/definitions/RunningState"
+        },
+        "provisioningState": {
+          "description": "Provisioning state of Managed CCF Resource",
+          "readOnly": true,
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "nodeCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of CCF nodes in the Managed CCF.",
+          "$ref": "#/definitions/NodeCount"
+        }
+      }
+    },
+    "ManagedCCF": {
+      "type": "object",
+      "description": "Managed CCF. Contains the properties of Managed CCF Resource.",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Properties of Managed CCF Resource.",
+          "$ref": "#/definitions/ManagedCCFProperties"
+        }
+      }
+    },
+    "ManagedCCFList": {
+      "description": "Object that includes an array of Managed CCF and a possible link for next set.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of Managed CCF",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedCCF"
+          }
+        },
+        "nextLink": {
+          "description": "The URL the client should use to fetch the next page (per server side paging).",
+          "type": "string"
+        }
+      }
+    },
+    "ManagedCCFBackup": {
+      "description": "Object representing Backup properties of a Managed CCF Resource.",
+      "type": "object",
+      "properties": {
+        "restoreRegion": {
+          "type": "string",
+          "readOnly": false,
+          "description": "The region where the backup of the managed CCF resource will eventually be restored to."
+        },
+        "uri": {
```

Comment:
>uri [](http://example.com/codeflow?start=9&length=3)

As this looks like SAS URI, so this should be marked as a secret: [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret).

Reply:
Added